### ISSUE TITLE
Add the ability to opt-out mounting ~/.ssh inside the execution environment

### ISSUE THREAD

**Issue by fridim** [ansible/ansible-runner#1194]
Is it possible to prevent ansible-runner from mounting user's `~/.ssh` into the container when using EEs?

Looking at https://github.com/ansible/ansible-runner/blob/devel/ansible_runner/utils/__init__.py#L475-L482, it seems it's currently not possible to configure ansible-runner to not mount host `~/.ssh`.

**Feature request:**  add a setting and an environment variable to disable mounting `~/.ssh/` entirely.

The documentation in https://ansible-navigator.readthedocs.io/en/latest/faq/#ssh-keys should probably be updated if this feature is implemented.

**Comment by tonykay** [comment]
I agree, an option. Our EEs in my opinion do need to default to their "default" or expected behavior but this is a good option IMO.

**Comment by gundalow** [comment]
@fridim @tonykay Thanks for your comments here. 
Would you please be able to detail your setup/use-case here so we can better understand your request? Thanks

**Comment by Shrews** [comment]
Per an out-of-band conversation, if a non-SSH connection plugin is being used for node communication, mounting the SSH directory seems useless. I'd be on board with such an enhancement.
